### PR TITLE
fix #274925: Unable to select text in Chord Symbol

### PR DESCRIPTION
**Pull request by handrok**
Add correct placement for _layout field in Harmony class.

### CODE REVIEW

File: libmscore/harmony.cpp [hunk 1]
```diff
@@ -1085,6 +1087,14 @@ void Harmony::calculateBoundingRect()
                   }
             setbbox(bb);
             setbboxtight(tbb);
+
+            for (int i = 0; i < rows(); ++i) {
+                  TextBlock* t = &textBlockList()[i];
```
Comment (anatoly-os) [inline libmscore/harmony.cpp]: Why not to use ref like:
```
TextBlock& t = textBlockList()[i];
t.layout(this);
for (auto& s : t.fragments()) {
...
```

**Comment by anatoly-os** [comment]
@wschweer could you please review the code?
I'm not sure I understand whether we need this https://github.com/musescore/MuseScore/pull/3957/files#diff-26d27ddfb560964c358996e7381459ffR1095 and why. @handrok could you please comment and put the comment in the code?

**Comment by handrok** [comment]
I put 
   TextBlock& t = textBlockList()[i];
   t.layout(this) 
Because there is ambiguity. Look, when MS switch to editing Harmony MS draws text from textBlockList(). When MS switches back to normal state it draws text from textList which is private field of Harmony class. I call t.layout(this)  to define correct position of text in editing mode

**Comment by anatoly-os** [comment]
@handrok could you please mention the reason why you introduced the `_needAutoplace`? Also, please mention that this is workaround for real issue arised from the fact that `chordSymbol` position and size are calculated incorrectly.

**Comment by anatoly-os** [comment]
@handrok could you please rebase the PR and check whether the fix is still actual?

**Comment by handrok** [comment]
@anatoly-os done

**Comment by handrok** [comment]
@anatoly-os done

**Pull request merged**